examples/core_api/endpoints/test_endpoint.rb

```diff
@@ -13,7 +13,9 @@ class TestEndpoint < Apia::Endpoint
         description "Any string will do, it's not validated"
       end
 
-      field :time, type: Objects::Time, include: "unix,day_of_week,year[as_string]", null: true do
+      field :time,
+            type: Objects::Time,
+            include: "unix,day_of_week,year[*,-as_integer,-as_array_of_strings,-as_array_of_enums]", null: true do
```

Comment:
I changed this to make sure we handle negative declarations in the `include` string